python/hsfs/engine/spark.py

```diff
@@ -17,6 +17,8 @@
 import pandas as pd
 import numpy as np
 
+import os
```

Comment:
nitpick, standard library imports go to the top

Reply:
Fixed